### DISCUSSION TITLE
[QUESTION] How to download all my saved posts from a .cvs file?

### DISCUSSION THREAD

**Discussion by LEARJS** [Serene-Arc/bulk-downloader-for-reddit#929] [Q&A]
Since the saved command can only pull the last 1000 saved posts, I requested all my data from Reddit but I'm unable to get it to download all the links from the .cvs file.

**Comment by Fakeaccount12312** [comment]
This question has already been asked, I aswered it here:
https://github.com/aliparlakci/bulk-downloader-for-reddit/discussions/877#discussioncomment-6144696
As stated there, you can only download posts this way, as #851 still isn't fixed.
